Tools/autotest/unittest/extract_param_defaults_unittest.py

```diff
@@ -10,7 +10,7 @@
 
 import unittest
 from unittest.mock import patch, MagicMock
-from extract_param_defaults import extract_parameter_default_values, missionplanner_sort, \
+from extract_param_defaults import extract_parameter_values, missionplanner_sort, \
```

Comment:
Presumably we're missing a test for this new functionality here...

Reply:
Yes, I did not add any new tests for the new functionality yet.